server/routes/sentry/tasks.py

```
@@ -0,0 +1,420 @@
+
+    triggered_rule = data.get("triggered_rule")
+    if triggered_rule:
+        meta["triggeredRule"] = triggered_rule
+
+    return {k: v for k, v in meta.items() if v is not None}
+
+
+@celery_app.task(
+    bind=True, max_retries=3, default_retry_delay=30, name="sentry.process_event"
+)
+def process_sentry_event(
```

Ruff (0.15.12)

[warning] Too many statements (72 > 50)

(PLR0915)